tests/test_app.py

```diff
@@ -683,11 +763,16 @@ def test_networks_remove(site_client, network):
     """Test ``nsot networks remove``."""
     runner = CliRunner(site_client.config)
     with runner.isolated_filesystem():
-        # Just delete the network we have.
+        # Just delete the network we have by id.
         result = runner.run('networks remove -i %s' % network['id'])
         assert result.exit_code == 0
         assert 'Removed network!' in result.output
 
+        # Create a new network and then delete it by CIDR.
+        runner.run('networks add -c 10.20.30.0/24')
+        result = runner.run('networks remove -i 10.20.30.0/24')
+        assert_output(result, ['Removed network!'])
```

Comment:
Possibly not appropriate for this set of changes, but I've often wondered about making these output messages more meaningful/explicit.